build.gradle

```diff
@@ -105,6 +105,7 @@ pmd {
   sourceSets = [sourceSets.main, sourceSets.test, sourceSets.functionalTest, sourceSets.integrationTest, sourceSets.smokeTest]
   reportsDir = file("$project.buildDir/reports/pmd")
   ruleSetFiles = files("config/pmd/ruleset.xml")
+  ruleSets = []
```

Comment:
https://discuss.gradle.org/t/pmd-plugin-reporting-violation-of-unconfigured-rule-with-gradle-5/29959